featurestore/components/feature_retrieval/spec.yaml

```diff
@@ -0,0 +1,62 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+
+name: feature_retrieval
+version: 1
+type: spark
+
+display_name: Feature Retrieval
+description: Retrieval component to be used to retrieve features from feature store.
+is_deterministic: true
+
+code: ../src
+entry:
+  file: retrieval.py
+
+inputs:
```

Comment:
Would you please add descriptions to inputs and outputs?

Reply:
Added descriptions